### ISSUE TITLE
The search doesnt seem to work anymore

### ISSUE THREAD

**Issue by JeffThunder** [MegaMoo77/favorites-search#16]
Hi, first I'd like to thank you for the tool I've been using it for months and it work great. However, recently the tool doesn't show suggest tags like it used to and the tool haven't been able to work for me. Is there a way to fix this?

**Comment by DownToBius** [comment]
Hey mate. I'm a user like you. The tool has been blocked again by R34, hard this time.

I don't know if MM is looking for alternatives or not.

**Comment by joe-smith1999** [comment]
> Hi, first I'd like to thank you for the tool I've been using it for months and it work great. However, recently the tool doesn't show suggest tags like it used to and the tool haven't been able to work for me. Is there a way to fix this?

yeah, for whatever reason the r34 admins blocked this from working. you might be able to get it working on your own computer, but I can't even begin to understand how to go about that. MegaMoo is probably working on making a local version that's easy to run, but that seems like something that would take a while.

fuck the mods on that website man, seriously lmao.

**Comment by DownToBius** [comment]
Anyone got any news?

**Comment by Suntaun** [comment]
It would be nice to have an F95 or discord for this tool and ones like it.

**Comment by Librake** [comment]
Good news, mates! 
After waiting for several months like all of you, I decided to write my own search script. And it's almost ready, I think I'll be able to publish it in the next few days.

**Comment by Suntaun** [comment]
> Good news, mates! After waiting for several months like all of you, I decided to write my own search script. And it's almost ready, I think I'll be able to publish it in the next few days.

I'm very glad to see someone else making a favorite search tool. Eagerly await your project release.

**Comment by DownToBius** [comment]
> Good news, mates! After waiting for several months like all of you, I decided to write my own search script. And it's almost ready, I think I'll be able to publish it in the next few days.

Looking forward to it. Is it going to available as a website like MM's was?

**Comment by Librake** [comment]
>Is it going to available as a website like MM's was?

As a userscript, you should install Tampermonkey browser extension and then add my script when it becomes available.
It will expand R34 site by adding search bar on your favorites page.

**Comment by DownToBius** [comment]
> As a userscript, you should install Tampermonkey browser extension and then add my script when it becomes available. It will expand R34 site by adding search bar on your favorites page.

Easy. I can get TM on Firefox Mobile, will it support mobile format?

Also curious what features you've got. MM had an original image link for example, that was nice.

**Comment by Librake** [comment]
> I can get TM on Firefox Mobile, will it support mobile format?

Hmm, I should check this out. I think it'll work, but needs some ui fixes to display correctly on mobile devices.

>Also curious what features you've got. MM had an original image link for example

Yeah this one and some more. After release, it would be nice to get feedback to add more improvements and features.

**Comment by DownToBius** [comment]
> Yeah this one and some more. After release, it would be nice to get feedback to add more improvements and features.

Sounds good, if you can, add a little button or link on the favs page that links to your github

**Comment by Librake** [comment]
I have created the [Discord Server](https://discord.gg/jZzYFNeCTw) for further discussion and updates. I think it's right to stop spamming under someone else's project.
